chain/penumbra/penumbra_chain.go

```diff
@@ -109,56 +104,65 @@ func (c *PenumbraChain) Initialize(ctx context.Context, testName string, cli *cl
 
 // Exec implements chain interface.
 func (c *PenumbraChain) Exec(ctx context.Context, cmd []string, env []string) (stdout, stderr []byte, err error) {
-	return c.getRelayerNode().PenumbraAppNode.Exec(ctx, cmd, env)
+	return c.getFullNode().PenumbraAppNode.Exec(ctx, cmd, env)
 }
 
-func (c *PenumbraChain) getRelayerNode() PenumbraNode {
-	if len(c.PenumbraNodes) > c.numValidators {
-		// use first full node
-		return c.PenumbraNodes[c.numValidators]
-	}
+func (c *PenumbraChain) getFullNode() *PenumbraNode {
 	// use first validator
 	return c.PenumbraNodes[0]
 }
 
 // Implements Chain interface
 func (c *PenumbraChain) GetRPCAddress() string {
-	return fmt.Sprintf("http://%s:26657", c.getRelayerNode().TendermintNode.HostName())
+	return fmt.Sprintf("http://%s:26657", c.getFullNode().TendermintNode.HostName())
 }
 
 // Implements Chain interface
 func (c *PenumbraChain) GetGRPCAddress() string {
-	return fmt.Sprintf("%s:9090", c.getRelayerNode().TendermintNode.HostName())
+	return fmt.Sprintf("%s:9090", c.getFullNode().TendermintNode.HostName())
 }
 
 // GetHostRPCAddress returns the address of the RPC server accessible by the host.
 // This will not return a valid address until the chain has been started.
 func (c *PenumbraChain) GetHostRPCAddress() string {
-	return "http://" + c.getRelayerNode().PenumbraAppNode.hostRPCPort
+	return "http://" + c.getFullNode().PenumbraAppNode.hostRPCPort
 }
 
 // GetHostGRPCAddress returns the address of the gRPC server accessible by the host.
 // This will not return a valid address until the chain has been started.
 func (c *PenumbraChain) GetHostGRPCAddress() string {
-	return c.getRelayerNode().PenumbraAppNode.hostGRPCPort
+	return c.getFullNode().PenumbraAppNode.hostGRPCPort
 }
 
 func (c *PenumbraChain) HomeDir() string {
 	panic(errors.New("HomeDir not implemented yet"))
```

Comment:
HomeDir() can now use - `return c.getFullNode().PenumbraAppNode.HomeDir()`

Reply:
good catch! i'll update this